vendor/active-docs/app/assets/javascripts/active-docs/form_handler.js.erb

```diff
@@ -333,8 +333,6 @@
 
   function getHost() {
     var base = basePathComponents();
-    // This is to make IE happy.
-    // https://connect.microsoft.com/IE/feedback/details/817343/ie11-scripting-value-of-htmlanchorelement-host-differs-between-script-created-link-and-link-from-document
```

Comment:
Comment removed but no code change. Was this intentional?

Reply:
Yes. The link doesn't exist anymore and I don't think it's worth investigating.

Reply:
I'll take a look, thanks @mayorova !